setup.py

```diff
@@ -12,4 +18,10 @@
     keywords=['computer vision', 'image processing', 'opencv', 'matplotlib'],
     classifiers=[],
     scripts=['bin/range-detector'],
+    install_requires=[
+        'numpy', 
+        'scipy', 
```

Comment:
I have raised a PR to remove scipy dependency. If that gets approved then you might have one less dependency to install.

https://github.com/PyImageSearch/imutils/pull/277#issue-1437785798

Reply:
@farqis When your PR is merged, please update that here and resolve this conversation.

Reply:
@farqis Was your PR merged (for removing scipy dependency)?